README.md

```diff
@@ -68,6 +68,8 @@ The extension currently has two settings:
 The plugin uses the Python extension to change environments. This can be done by clicking on the active environment in the bottombar
 and then selecting the environment to activate.
 
+If the environment is not shown in the bottom bar, change the setting "python.interpreter.infoVisibility" to "Always".
```

Comment:
Can't we just set this? or is that considered impolite?

Reply:
I don't think I'd like it if a plugin would alter my settings. I would get frustrated If I change the setting and for some reason my changes are always overwritten and I wouldn't know why this is happening.
If we still decide to do this, I propose to change the setting on activation, and set it back to the previous value on deactivation. 
I haven't verified (yet) if it is doable but I suppose it is.

Reply:
@wouterdb Would you want me to try?